### PR TITLE
List consultee responses if any exist

### PR DESCRIPTION
**Pull request by benjamineskola**
### Description of change

If any consultee responses exist with a response text attached, list them for ease of writing a summary.

### Story Link

https://trello.com/c/KYztoXq0/1888-view-all-comments-received-from-consultees

### Screenshots

Screenshot 2023-09-11 at 15 21 41

### CODE REVIEW

File: app/views/planning_application/assessment_details/consultation_summary/_information.html.erb [hunk 1]
```diff
@@ -6,3 +6,24 @@
     read_only: read_only
   }
 ) %>
+
+<% if @planning_application.consultees.where.not(response: nil).present? %>
```
Comment (benbaumann95) [inline app/views/planning_application/assessment_details/consultation_summary/_information.html.erb:10]: Can we add this as a scope on the consultee model i.e 

`scope :with_response, -> { where.not(response: nil) }`

guess you can even store this as an instance variable if calling multiple times

File: app/views/planning_application/assessment_details/consultation_summary/_information.html.erb [hunk 2]
```diff
@@ -6,3 +6,24 @@
...
+  <h3>Consultation responses</h3>
+  <div class="govuk-accordion" data-module="govuk-accordion">
+    <% @planning_application.consultees.where.not(response: nil).each do |consultee| %>
```
Comment (benbaumann95) [inline app/views/planning_application/assessment_details/consultation_summary/_information.html.erb:13]: same as above

**Review by benbaumann95 — COMMENTED** [review]
Could we also add a spec for this pls, thanks

**Comment by benbaumann95** [comment]
> Could we also add a spec for this pls, thanks

Have the POs suggested that this was all that was required for this task?

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**